### ISSUE TITLE
Implement sign up form in footer

### ISSUE THREAD

**Issue by gsidebo** [mitodl/mitxonline#20]
Depends on #18 

Like in mitxpro, we want to implement a sign up form for users to opt into an mitxonline mailing list. Part of this issue will involve some coordination with Peter/Ferdi/devops to get the correct settings in place so that these form submissions go to the right destination.

**Acceptance criteria:**
- Sign up form added to the footer
- Submitting the sign up form adds the user to the relevant mailing list
- Entering a blank or invalid email address triggers a validation message (this is implemented in xpro as well)
- Issue(s) created to add the correct settings for the mailing list in deployed environments (CI, RC, prod)

**Comment by gsidebo** [comment]
On hold until we determine which service we're going to use for the mailing list
